Sources/Controllers/2.0/OACustomSourceDetailsViewController.mm

```diff
@@ -277,6 +320,15 @@ - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForR
     return nil;
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSDictionary *item = _data[indexPath.row];
+    NSString *type = item[@"type"];
+    if ([type isEqualToString:[OAImagesTableViewCell getCellIdentifier]])
```

Comment:
I think there's no need to make OAImagesTableViewCell dynamic, please add a height constraint and do not change the height while scrolling images